tools/mount_gcsfuse/main.go

```diff
@@ -83,6 +83,7 @@ func makeGcsfuseArgs(
 		case "implicit_dirs",
 			"disable_http2",
 			"experimental_local_file_cache",
+			"experimental_enable_storage_client_library",
```

Comment:
Can you create a bug to add a g3doc mentionning what needs to be done when a new flag is added